x-pack/plugins/threat_intelligence/public/modules/block_list/containers/flyout.tsx

```diff
@@ -58,7 +60,9 @@ export const BlockListFlyout: VFC<BlockListFlyoutProps> = ({ indicatorFileHash }
     item,
     policies: [],
     FormComponent,
-    onClose: () => {},
+    onClose: () => {
+      setBlockListIndicatorValue('');
```

Comment:
maybe that context could export this as a separate method, such as `clearBlockListIndicatorValue` - that way you could probably just do something like `onClose: clearBlockListIndicatorValue`